var/spack/repos/builtin/packages/py-beancount/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBeancount(PythonPackage):
+    """A double-entry bookkeeping computer language that lets you define
+       financial transaction records in a text file, read them in memory,
+       generate a variety of reports from them, and provides a web
+       interface.."""
+
+    homepage = "http://furius.ca/beancount/"
+    pypi     = "https://pypi.io/packages/source/b/beancount/beancount-2.3.3.tar.gz"
```

Comment:
```suggestion
    pypi     = "beancount/beancount-2.3.3.tar.gz"
```

Reply:
Drats, sorry that I forgot to change this.